recipes/eastl/all/conanfile.py

```diff
@@ -25,92 +40,107 @@ class EastlConan(ConanFile):
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
     @property
-    def _minimum_cpp_standard(self):
+    def _min_cppstd(self):
         return 14
 
     @property
-    def _minimum_compilers_version(self):
+    def _compilers_minimum_version(self):
         return {
             "Visual Studio": "14",
-            "gcc": "5",
-            "clang": "3.2",
-            "apple-clang": "4.3",
+            "gcc": "6",
+            "clang": "6",
+            "apple-clang": "6",
         }
 
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("eabase/2.09.06")
+        self.requires("eabase/2.09.12")
+
+    def package_id(self):
+        self.info.clear()
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
-
-        mininum_compiler_version = self._minimum_compilers_version.get(str(self.settings.compiler))
-        if mininum_compiler_version and tools.Version(self.settings.compiler.version) < mininum_compiler_version:
-            raise ConanInvalidConfiguration("Compiler is too old for c++ {}".format(self._minimum_cpp_standard))
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["EASTL_BUILD_BENCHMARK"] = False
-        self._cmake.definitions["EASTL_BUILD_TESTS"] = False
-        self._cmake.configure(build_folder=self._build_subfolder)
-        return self._cmake
+        get(
+            self,
+            **self.conan_data["sources"][self.version],
+            destination=self.source_folder,
+            strip_root=True,
+        )
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["EASTL_BUILD_BENCHMARK"] = False
+        tc.variables["EASTL_BUILD_TESTS"] = False
```

Comment:
```suggestion
        tc.variables["EASTL_BUILD_TESTS"] = False
        tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

taking a look in the CMakeLists.txt from eastl project, it does not declare a required C++ standard, but headers consumes C++11 and C++14 features. A minimal standard should be declared.